APITests/test_resources_utils.py

```diff
@@ -102,6 +102,16 @@ def create_all_basic_resources(
             entity_view_name=entity_view_name,
         )
         return project, project_id, data_folder, entity_view
+    
+    def delete_resource(self, resource_id: str) -> None:
```

Comment:
why there are two `delete_resource` function?

Reply:
Since the `CreateTestFiles` class also creates synapse resources I thought it necessary to also have a mechanism to clean them up @linglp

Reply:
Thanks for the comment. Could you create a separate class for this? : 
```
class SynapseUtils:
    @staticmethod
    def delete_resource(syn: Synapse, resource_id: str) -> None:
        """Delete a resource from Synapse."""
        syn.delete(resource_id)
```

Reply:
@linglp yeah sure; could you explain why you feel the delete method is better suited to a separate class?

Reply:
Yeah. I think since the deleting resources function will be used by two classes, it is better that this gets extracted into its own class.

Reply:
Sure, I can separate them